dev-resources/less/magic-bootstrap.less

```diff
@@ -167,7 +167,7 @@
 @nav-tabs-justified-link-border-color: @brand-primary;
 @nav-tabs-justified-active-link-border-color: @body-bg;
 @nav-pills-border-radius: @border-radius-base;
-@nav-pills-active-link-hover-bg: @component-active-bg;
+@nav-pills-active-link-hover-bg: @sininen4;
```

Comment:
Tää on definattu näin: 
@component-active-bg: @sininen5;

Vaihda mielummin toi muuttuja -> @sininen4, äläkä itse navia

Reply:
Laitetaan tällä menetelmällä. Tälleen meinasin aiemmin tehdä, mutta tää vaikuttaa joihinkin muihin kohtiin jota en tiedä onko haluttu vaikutus.